std/algorithm/comparison.d

```diff
@@ -546,6 +544,18 @@ elements.
 
 If the ranges are strings, $(D cmp) performs UTF decoding
 appropriately and compares the ranges one code point at a time.
+
+Params:
+    pred = The predicate used for comparison.
+    r1 = The first range.
+    r2 = The second range.
+
+Returns:
+    0 if both ranges compare equal. -1 if the first differing element of $(D
```

Comment:
compare equal *according to `pred`*. IMO, it looks a little confusing when the following two sentences are explicit about it.